src/components/TileCatalogNew/TileCatalogNew.test.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { render, fireEvent, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
let's be sure to double check whether we need to pull this in for each of these test.

Reply:
https://github.com/IBM/carbon-addons-iot-react/issues/1400